docs/agile/scrum.md

```diff
@@ -0,0 +1,48 @@
+# Scrum
+
+## Sprint name
+Instead of using sprint names like Sprint 1, Sprint 2, and so on try to name 2022 Q4 Sprint 1, 2022 Q4 Sprint 2
+
+### Why?
+- It is easier to analyze your work after generated reports for example
```

Comment:
It may be worth adding that this is also good practice from a business perspective. 

In the organisation where I work, synchronisation of the planning and release process between multiple departments is also required. So the naming of sprints needs to reflect that too.